llvm/test/CodeGen/NVPTX/truncate_zext.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc -march=nvptx64 < %s | FileCheck %s
```

Comment:
You may want to use `llvm/utils/update_llc_test_checks.py` to generate the checks.